src/pybind/mgr/dashboard/ci/cephadm/quick-bootstrap.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+source bootstrap-cluster.sh > /dev/null 2>&1
+
+set +x
+
+show_help() {
+  echo "Usage: ./quick-bootstrap.sh [OPTIONS]"
+  echo ""
+  echo "Options:"
+  echo "  -u, --use-cached-image     Uses the existing podman image in local. Only use this if there is such an image present."
+  echo "  -dir, --ceph-dir             Use this to provide the local ceph directory. eg. --ceph-dir=/path/to/ceph"
+  echo "  -e, --expanded-cluster     To add all the hosts and deploy OSDs on top of it."
+  echo "  -h, --help             Display this help message."
+  echo ""
+  echo "Example:"
+  echo "  ./quick-bootstrap.sh --use-cached-image"
+}
+
+use_cached_image=false
+expanded_cluster=false
+
+for arg in "$@"; do
+  case "$arg" in
+    -u|--use-cached-image)
+      use_cached_image=true
+      ;;
+    -dir=*|--ceph-dir=*)
+      ceph_dev_folder=${arg#*=}
+      ;;
+    -e|--expanded-cluster)
+      expanded_cluster=true
```

Comment:
probably that involves changes in the [bootstrap-cluster.sh](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/ci/cephadm/bootstrap-cluster.sh#L39) file itself because as it is right now if you provide the `expanded_cluster` its going to deploy a fully functional expanded cluster with osds on it. 

If we want, later on we can introduce flags like `--deploy-all-hosts`, `--deploy-rgw` and so on. Or even look into something like the vstart does. `MGR=2 RGW=1 quick-bootstrap.sh`.

Reply:
Okay even if I dont pass this flag I am getting all osds scheduled.
And after looking I see the bootsrap.sh just check whether this flag is defined, ehnce even if we pass `false` it goes on creating.

So we need to update that or not pass this option if falsy.